pre-commit.sh

```diff
@@ -1,5 +1,39 @@
-#!/bin/bash -eux
-yapf --in-place --recursive .
+#!/bin/bash -ux
+
+base_remote="$(1:-origin)"
+base_branch="$(1:-master)"
+base_remote_branch="$(1:-master)"
+
+set +e
+
+if [ "$(<.git/refs/heads/${base_branch})" != "$(<.git/refs/remotes/${base_remote}/${base_remote_branch})" ]; then
```

Comment:
with uninitialized variables `base_branch`, `base_remote` and `base_remote_branch it always returns false